modules/exploits/linux/local/pkexec.rb

```diff
@@ -66,6 +66,20 @@ def executable_path
     @executable_path
   end
 
+  def check
+    version = cmd_exec('pkexec --version')
+    if version.empty?
```

Comment:
Also, it won't be an empty string if ```pkexec``` isn't found. It *could* be, but ```stderr``` is captured, too: ```/bin/sh: 3: pkexec: not found```. Any chance you can revert/reset so I can PR to you?